tests/phpunit/tests/Checker/Checks/File_Type_Check_Tests.php

```diff
@@ -98,4 +98,23 @@ public function test_run_without_any_file_type_errors() {
 		$this->assertEmpty( $errors );
 		$this->assertSame( 0, $check_result->get_error_count() );
 	}
+
+	public function test_run_with_badly_named_errors() {
```

Comment:
May be we should add more tests for other types of errors like directory name with space, file name with restricted characters, etc.

Reply:
Ok, I agree. I'll go for it.